### PR TITLE
update the ipv6 documentation

### PR DESCRIPTION
**Pull request by szaimen**
Close https://github.com/nextcloud/all-in-one/issues/4887

I've also already updated the docs in https://github.com/nextcloud/all-in-one/discussions/2045

### CODE REVIEW

**Comment by Zoey2936** [comment]
network mode bridge should be added to the compose.yaml, because otherwise docker compose will create a new network just for the mastercontainer, if we add network mode bridge the mastercontainer would be added to the bridge network, like it alsp happens when the mastercontainer is created using the docker run command.
Also the issue in the mentioned discussion was closed: https://github.com/moby/moby/issues/41651 https://github.com/nextcloud/all-in-one/discussions/2557

**Comment by Zoey2936** [comment]
And I think instead of #2045 the discussion #2557 should be mentioned

**Comment by szaimen** [comment]
Can you maybe make some suggestions to the code or commit your ideas directly?

**Comment by Zoey2936** [comment]
not possible because of deleted lines or unchanged lines

**Comment by szaimen** [comment]
Committing to the branch is also not possible?

**Comment by Zoey2936** [comment]
done and #2557 should be rewritten since the issue was closed with docker v26

**Comment by szaimen** [comment]
> done 

Thanks :)

> and #2557 should be rewritten since the issue was closed with docker v26

What exactly do I need to change there?

**Comment by Zoey2936** [comment]
> > and #2557 should be rewritten since the issue was closed with docker v26
> 
> What exactly do I need to change there?

the mentioned issue was closed

**Comment by szaimen** [comment]
> > > and #2557 should be rewritten since the issue was closed with docker v26
> > 
> > 
> > What exactly do I need to change there?
> 
> the mentioned issue was closed

Yes it was closed but it doesnt look like the issue is fixed based on the conments?

**Comment by Zoey2936** [comment]
but they seem to talk about ipv6 only containers?

File: manual-install/update-yaml.sh [hunk 1]
```diff
@@ -139,16 +139,8 @@ done
 
 cat << NETWORK >> containers.yml
 
-# Inspired by https://github.com/mailcow/mailcow-dockerized/blob/master/docker-compose.yml
 networks:
   nextcloud-aio:
-    name: nextcloud-aio
```
Comment (Zoey2936) [inline manual-install/update-yaml.sh:145]: I think the name should stay

Comment (szaimen) [reply manual-install/update-yaml.sh:145]: done

**Comment by szaimen** [comment]
> > > > and #2557 should be rewritten since the issue was closed with docker v26
> > > 
> > > 
> > > What exactly do I need to change there?
> > 
> > 
> > the mentioned issue was closed
> 
> Yes it was closed but it doesnt look like the issue is fixed based on the conments?

I closed the discussion since after testing this a bit I actually cannot reproduce the problem anymore :)

**Comment by Zoey2936** [comment]
> > > > > and #2557 should be rewritten since the issue was closed with docker v26
> > > > 
> > > > 
> > > > What exactly do I need to change there?
> > > 
> > > 
> > > the mentioned issue was closed
> > 
> > 
> > Yes it was closed but it doesnt look like the issue is fixed based on the conments?
> 
> I closed the discussion since after testing this a bit I actually cannot reproduce the problem anymore :)

Can you then create a new discussion/wiki with a guide on how to recreate the nextcloud-aio network?

**Comment by szaimen** [comment]
> > > > > > and #2557 should be rewritten since the issue was closed with docker v26
> > > > > 
> > > > > 
> > > > > What exactly do I need to change there?
> > > > 
> > > > 
> > > > the mentioned issue was closed
> > > 
> > > 
> > > Yes it was closed but it doesnt look like the issue is fixed based on the conments?
> > 
> > 
> > I closed the discussion since after testing this a bit I actually cannot reproduce the problem anymore :)
> 
> Can you then create a new discussion/wiki with a guide on how to recreate the nextcloud-aio network?

done

**Review by Zoey2936 — APPROVED** [review]

**Comment by szaimen** [comment]
Thanks Zoey! :)

**Comment by Kirk1984** [comment]
i wanted to test this on a pi, but ipv6 ist still disabled. Docker 27.0.3, daemon.json is modified as documented, copy pasted new compose.yaml. inspecting the bridge network shows "nextcloud-aio-mastercontainer", but no ipv6 address is shown and the bridge network shows "EnableIPv6": false.

Am i missing something?

**Pull request merged**